bundle-size/action.yml

```diff
@@ -14,6 +14,14 @@ inputs:
     description: "Only post reports to PRs if the total entrypoint diff percentage is greater than this number."
     required: false
     default: "5"
+  working-directory:
```

Comment:
In IRM monorepo we want to run it in subdirectory, e.g. `packages/@plugins/grafana-oncall-app`